sdk/servicebus/azure-servicebus/setup.py

```diff
@@ -77,7 +77,7 @@
         'azure',
     ]),
     install_requires=[
-        'uamqp~=1.2.0',
+        'uamqp>=1.2.5',
```

Comment:
changelog update: Updated uAMQP dependency to 1.2.5

Reply:
Added++